scripts/zkai_native_seq32_attention_mlp_generated_adjacent_label_inventory_gate.py

```
@@ -0,0 +1,818 @@
+)
+EXPECTED_CLI_COMMANDS = (
+    "build-input-rmsnorm-fused-adjacent",
+    "build-input-rmsnorm-fused-adjacent-label-probe-a",
+    "build-input-rmsnorm-fused-adjacent-label-probe-b",
+)
+ADAPTER_MODE_TO_LABEL_ID = {
+    "rmsnorm_input_fused_adjacent_fixed_v1": "fixed_adjacent_layout",
+    "rmsnorm_input_fused_adjacent_label_probe_a_v1": "adjacent_label_probe_a",
+    "rmsnorm_input_fused_adjacent_label_probe_b_v1": "adjacent_label_probe_b",
+}
+ADAPTER_MODE_TO_CLI_COMMAND = dict(zip(EXPECTED_ADJACENT_MODES, EXPECTED_CLI_COMMANDS))
```

Ruff (0.15.13)

[warning] `zip()` without an explicit `strict=` parameter

Add explicit value for parameter `strict=`

(B905)